extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -172,7 +172,8 @@ public function getMobileWikiVariables() {
 		global $wgCityId, $wgStyleVersion, $wgContLang, $wgContentNamespaces, $wgDefaultSkin, $wgCdnRootUrl,
 		       $wgRecommendedVideoABTestPlaylist, $wgFandomAppSmartBannerText, $wgTwitterAccount,
 		       $wgEnableFeedsAndPostsExt, $wgEnableEmbeddedFeeds, $wgDevelEnvironment, $wgQualarooDevUrl, $wgQualarooUrl,
-		       $wgRightsText, $wgRightsUrl, $wgRedirectFilePagesForAnons, $wgEnableDiscussions;
+		       $wgRightsText, $wgRightsUrl, $wgRedirectFilePagesForAnons, $wgEnableDiscussions,
+		       $wgAffiliateUnitEnabledDate;
```

Comment:
what about `wgWatchShowGeos` and `wgWatchShowEnabledDate`? according to PR description both of them should be here as well

Reply:
@rybmat Those are exported via hook with this: https://github.com/Wikia/app/blob/cake-5230/extensions/wikia/WatchShow/WatchShowHooks.class.php#L11-L12